spec/facility_spec.rb

```diff
@@ -2,25 +2,150 @@
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+  describe '#register_vehicle' do
+    it 'registers vehicles if capable' do
+      expect(@facility_1.register_vehicle(@bolt)).to eq(nil)
+      @facility_1.add_service("Vehicle Registration")
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq(100)
+      expect(@facility_1.instance_variable_get(:@registered_vehicles).length).to eq(1)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@cruz.registration_date).to eq(Date.today)
+      @facility_1.register_vehicle(@camaro)
+      expect(@facility_1.collected_fees).to eq(125)
+      expect(@facility_1.instance_variable_get(:@registered_vehicles).length).to eq(2)
+      expect(@camaro.plate_type).to eq(:antique)
+      expect(@camaro.registration_date).to eq(Date.today)
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.collected_fees).to eq(325)
+      expect(@facility_1.instance_variable_get(:@registered_vehicles).length).to eq(3)
+      expect(@bolt.plate_type).to eq(:ev)
+      expect(@bolt.registration_date).to eq(Date.today)
+
+    end
+
+    it 'tracks collected fees' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@facility_1.collected_fees).to eq(100)
+      @facility_1.register_vehicle(@camaro)
+      expect(@facility_1.collected_fees).to eq(125)
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.collected_fees).to eq(325)
+    end
+
+    it 'stores registered vehicles' do
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      @facility_1.register_vehicle(@camaro)
+      @facility_1.register_vehicle(@bolt)
+      expect(@facility_1.instance_variable_get(:@registered_vehicles).length).to eq(3)
+      expect(@facility_1.instance_variable_get(:@registered_vehicles)).to all be_a(Vehicle)
```

Comment:
`instance_variable_get` is great for testing ivars without exposing them to outside classes with something like an `attr_reader`. Since you do have an `attr_reader` in the `Facility` class for these, I think you could just call those attributes as methods in the test
```
expect(@facility_1.registered_vehicles).to all be_a(Vehicle)
```